CityOfDetroit/bloom
# Feature/region enum

## Issue

- Closes #1097
- Addresses #1097

## Description

Adds a "region" field to the listing form.

## How Can This Be Tested/Reviewed?

1. Open partners portal
2. Fill the "region" field
3. Save listing
4. Check if the "region" has a proper value

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
